portfolio/src/main/java/com/google/sps/servlets/Server2.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+
+import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/head")
```

Comment:
What does head mean? Having a descriptive endpoint helps with reading both this Java code and the html or JavaScript that calls this endpoint.